docs/Terminology.rst

```diff
@@ -0,0 +1,25 @@
+.. _Observation_Terminology:
+
+***************
+Terminology
+***************
+
+Scheduling
+==================
+Determined during the python in astronomy conference:
+
+* **observing block** (OB): the minimum schedulable block
+* **priority**: number assigned to the **observing block** by the user (or TAC) that 
```

Comment:
@ejeschke Is this how we settled on using "priority"? My intuition is that "priority" is user-assigned and "rank" is TAC-assigned.